client/packages/inventory/src/Stocktake/DetailView/ContentArea/columns.ts

```diff
@@ -99,6 +99,7 @@ export const useStocktakeColumns = ({
           accessor: ({ rowData }) => {
             return rowData.item?.unitName ?? '';
           },
+          sortable: false,
```

Comment:
we don't support this in the api, so I've had to remove, along with those below

Reply:
Maybe should raise and issue for resolving these, or at least review is any are needed? How do we identify if this could be needed by customers?

Reply:
I'm sure that they are useful, and if we asked the stakeholders they'd request for them to stay sortable.. rather than assume, I'll make an issue 🙏